aiohttp/web_response.py

```diff
@@ -155,15 +156,14 @@ def set_status(
         self,
         status: int,
         reason: Optional[str] = None,
-        _RESPONSES: Mapping[int, Tuple[str, str]] = RESPONSES,
     ) -> None:
         assert not self.prepared, (
             "Cannot change the response status code after " "the headers have been sent"
         )
         self._status = int(status)
         if reason is None:
             try:
-                reason = _RESPONSES[self._status][0]
+                reason = HTTPStatus(self._status).phrase
             except Exception:
```

Comment:
```suggestion
            except ValueError:
```
I can't see any other exception that would be raised here.